src/main/java/duke/storage/ExportCsv.java

```diff
@@ -49,7 +49,8 @@ public static void exportLockers(List<Locker> lockerList) throws DukeException {
                 details[THIRD_COLUMN] = l.getZone().getZone();
                 details[FORTH_COLUMN] = l.getTag().getTagName();
 
-                if (details[THIRD_COLUMN].equals("in-use")) {
+                if (details[FORTH_COLUMN].equals("in-use")) {
+                    
                     details[FIFTH_COLUMN] = l.getUsage().get().getStudent().getName().getName();
                     details[SIXTH_COLUMN] = l.getUsage().get().getStudent().getMatricNumber().getMatricId();
                     details[SEVENTH_COLUMN] = l.getUsage().get().getStudent().getMajor().getCourse();
```

Comment:
how about exporting rental dates also? or like no-need for that?

Reply:
will try doing it....